### ISSUE TITLE
Shadows on newly rendered map

### ISSUE THREAD

**Issue by Uripura** [overviewer/Minecraft-Overviewer#1787]
Today I downloaded the newest Version for Debian and let it render an entire new map.
But now there are strange shadows on the map.

![Image of the shadow](https://uripura.de/img/share/2020-06-20_22-36_1910.png)

Exact Version I'm using:
Minecraft Overviewer 0.15.39 (98fe552)
built on Sat Jun 20 16:53:30 2020

At first I thought, that it had something to do with the librarys shiped with Debian, but the same issue pops up in the latest (0.15.39) Windows Version.

If I visit a affected Chunk the shadow goes away the next time I start a render.

**Comment by xFleury** [comment]
After updating Minecraft to 1.16 (the Nether update), my world is also generating weird shadows. I didn't have these strange shadows prior to the Nether update, so I think this is 1.16 related.

**Comment by CounterPillow** [comment]
Can you hand me a region file that contains a chunk that exhibits this problem? To be clear here, this is 100% Mojang's fault because we just render whatever lighting data they write, but maybe there's a way to detect this and fall back to having no lighting data for that chunk.

**Comment by endlisnis** [comment]
It's not just shadows, the top layer of blocks is skipped.  That might explain the shadows (since the lower blocks are underneath the top layer).

![image](https://user-images.githubusercontent.com/3651887/85738652-2fd18580-b6ce-11ea-9ea2-deae59647bb7.png)

**Comment by CounterPillow** [comment]
I can't do anything with screenshots. You're still not giving me region files to look at.

>That might explain the shadows (since the lower blocks are underneath the top layer).

Shadows are completely in the hands of minecraft, so overviewer skipping any part of anything doesn't affect them at all.

**Comment by cliffmeyers** [comment]
I see this lighting issue on my map based off a 1.15.2 world 

![lighting-issue](https://user-images.githubusercontent.com/143466/85758639-834bcf80-b6de-11ea-94d7-17b30122e926.jpg)

```
Minecraft Overviewer 0.15.2 (751ba39)
built on Sat Dec 28 08:42:33 2019
```

I noticed that these lines show up on the boundaries of chunks that were generated many versions ago (1.8?) with those generated more recently (1.15). I tried upgrading my entire world using the `--forceUpgrade` and `--eraseCache` flags on the world along with a full Overview render and they persist.

See file below, I used mcaselector to grab the chunks that should be affected.

[lighting-issue.zip](https://github.com/overviewer/Minecraft-Overviewer/files/4832565/lighting-issue.zip)

If that doesn't help, let me know and I grab the entire region file.
